tests/test_captions.py

```diff
@@ -0,0 +1,51 @@
+import pytest
+import json
+import os.path
+from requests.exceptions import HTTPError
+from src.rev_ai.models import Transcript, Monologue, Element
+from src.rev_ai.apiclient import RevAiAPIClient
+from tests.helpers.errors import get_error_test_cases
+
+try:
+    from urllib.parse import urljoin
+except ImportError:
+    from urlparse import urljoin
+JOB_ID = '1'
+URL = urljoin(RevAiAPIClient.base_url, 'jobs/{}/captions'.format(JOB_ID))
+
+
+@pytest.mark.usefixtures('mock_client', 'make_mock_response')
+class TestCaptionEndpoint():
+    def test_get_captions_with_success(self, mock_client, make_mock_response):
+        filename = 'TestFileToBeDeleted'
```

Comment:
For test variable values we like to use `example` where possible just to keep things consistent.